library/core/src/char/methods.rs

```diff
@@ -69,7 +69,7 @@ impl char {
     /// assert_eq!(char::from_u32(value_at_max + 1), None);
     /// ```
     #[stable(feature = "assoc_char_consts", since = "1.52.0")]
-    pub const MAX: char = '\u{10ffff}';
+    pub const MAX: char = '\u{10FFFF}';
```

Comment:
I'm okay with fixing this one because it's inconsistent, but the rest of them are self-consistent -- no leading zeroes.

Reply:
Understood. I will revert those, then.